WORKSPACE

```diff
@@ -1,7 +1,7 @@
 git_repository(
     name = "io_bazel_rules_go",
-    commit = "c7e02c851a8faf803f260800e7387248ccd47683",
-    remote = "https://github.com/bazelbuild/rules_go.git",
+    commit = "dbe525c13b24c8f0984f51351e63c0c3d774db59",
+    remote = "https://github.com/mikedanese/rules_go.git",
```

Comment:
This is temporary pending a change in rules_go.  I have done this on previous PRs that requires joint changes in rules_go and always revert before final merge.